### ISSUE TITLE
Intermediate-level guide in Time Series Forecasting example

### ISSUE THREAD

**Issue by chrisalexander** [tensorflow/tensorflow#45109]
## URL(s) with the issue:

https://github.com/tensorflow/docs/blob/master/site/en/tutorials/structured_data/time_series.ipynb

### Clear description

I believe this guide is overly complex for most basic time-series forecasting measures. Due to the introduction of a class for windowing, while this makes it easy to re-use code between examples, it doesn't do a very good job of creating basic windows that users might benefit from for a basic use case.

For example, I only want some basic windowing, I found more help setting this up in the keras preprocessing docs here https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing/timeseries_dataset_from_array than I did in this help page.

While this page demonstrates many potential use cases quite simply, I believe another page is required which outlines the very basic steps just to get one of these working with an intro to windowing as well.

**Comment by ymodak** [comment]
Perhaps this [lecture](https://www.coursera.org/lecture/tensorflow-sequences-time-series-and-prediction/preparing-features-and-labels-TYErD?utm_source=link&utm_medium=page_share&utm_content=vlp&utm_campaign=top_button) can help to get more clarity on setting up window sizes for time series data.

**Comment by chrisalexander** [comment]
Thanks, looks like it will be useful, I will review it in detail. However I feel the point still stands, it would be useful to have a simple-case code example in the docs. From my position, of having some experience with ML / neural nets but being new to tensorflow, it's more about understanding and adapting to the syntax and terminology rather than trying to learn the fundamentals, which I think this would help with.

Your call though :)

**Comment by Venkat6871** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The Tensorflow team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the Tensorflow space.
